llmcompare/question/question.py

```diff
@@ -581,3 +588,29 @@ def get_runner_input(self) -> list[dict]:
         for el in runner_input:
             el["top_logprobs"] = self.top_logprobs
         return runner_input
+
+
+class Logits(Question):
+    _runner_sampling_func_name = "specific_token_logits"
+
+    def __init__(
+        self,
+        tokens: list[int] | list[str],
```

Comment:
See this comment: https://github.com/johny-b/llmcompare/pull/9#discussion_r2191791262